Wrapping/Generators/Python/Tests/PythonTemplateTest.py

```diff
@@ -119,6 +119,17 @@
 median_kwarg = itk.MedianImageFilter.New(Input=reader.GetOutput())
 assert itk.class_(median) == itk.class_(median_kwarg)
 
+# filter type determined by the input passed as a primary input name input
+median_primary_kwarg = itk.MedianImageFilter.New(Primary=reader.GetOutput())
+assert itk.class_(median) == itk.class_(median_primary_kwarg)
+
+# First RequiredInputName: "FixedImage"
+fixed_image = reader.GetOutput()
+moving_image = fixed_image
+pde_registration = itk.PDEDeformableRegistrationFilter.New(
+    FixedImage=fixed_image, MovingImage=moving_image
+)
+
```

Comment:
Any clue what I'm doing wrong here?
```python
fixed_image = itk.Image[itk.F,2].New()
fixed_image.SetRegions([8,8])
fixed_image.AllocateInitialized()
moving_image = fixed_image
itk.pde_deformable_registration_filter(fixed_image, moving_image)
```
Produced:
```

Traceback (most recent call last):
  File "D:\src\ITK\Wrapping\Generators\Python\Tests\PythonTemplateTest.py", line 140, in <module>
    itk.pde_deformable_registration_filter(fixed_image, moving_image)
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\support\helpers.py", line 194, in image_filter_wrapper
    return image_filter(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\itkPDEDeformableRegistrationFilterPython.py", line 1543, in pde_deformable_registration_filter
    instance = itk.PDEDeformableRegistrationFilter.New(*args, **kwargs)
               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\support\template_class.py", line 750, in New
    return self[list(keys)[0]].New(*args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\itkPDEDeformableRegistrationFilterPython.py", line 341, in New
    template_class.New(obj, *args, **kargs)
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\support\template_class.py", line 815, in New
    itk.set_inputs(self, args, kargs)
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\support\extras.py", line 1622, in set_inputs
    new_itk_object.SetInput(args[0])
  File "C:\Users\user\.conda\envs\ITKSwigPy311\Lib\site-packages\itk\itkImageToImageFilterAPython.py", line 14039, in SetInput
    return _itkImageToImageFilterAPython.itkImageToImageFilterIVF22IVF22_SetInput(self, *args)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: Expecting argument of type itkImageVF22 or itkImageSourceIVF22.
Additional information:
Wrong number or type of arguments for overloaded function 'itkImageToImageFilterIVF22IVF22_SetInput'.
  Possible C/C++ prototypes are:
    itkImageToImageFilterIVF22IVF22::SetInput(itkImageVF22 const *)
    itkImageToImageFilterIVF22IVF22::SetInput(unsigned int,itkImageVF22 const *)
```

Reply:
pde_deformable_registration_filter might not be wrapped for F2?

Reply:
Nice guess, @dzenanz, but `print(itk.PDEDeformableRegistrationFilter.GetTypes())` says:

```
<itkTemplate itk::PDEDeformableRegistrationFilter>
Options:
  [<class 'itk.itkImagePython.itkImageF2'>, <class 'itk.itkImagePython.itkImageF2'>, <class 'itk.itkImagePython.itkImageVF22'>]
  [<class 'itk.itkImagePython.itkImageF3'>, <class 'itk.itkImagePython.itkImageF3'>, <class 'itk.itkImagePython.itkImageVF33'>]
  [<class 'itk.itkImagePython.itkImageSS2'>, <class 'itk.itkImagePython.itkImageSS2'>, <class 'itk.itkImagePython.itkImageVF22'>]
  [<class 'itk.itkImagePython.itkImageSS3'>, <class 'itk.itkImagePython.itkImageSS3'>, <class 'itk.itkImagePython.itkImageVF33'>]
  [<class 'itk.itkImagePython.itkImageUC2'>, <class 'itk.itkImagePython.itkImageUC2'>, <class 'itk.itkImagePython.itkImageVF22'>]
  [<class 'itk.itkImagePython.itkImageUC3'>, <class 'itk.itkImagePython.itkImageUC3'>, <class 'itk.itkImagePython.itkImageVF33'>]
```

So that should be fine for `itk.Image[itk.F,2]`, right?

Reply:
Oh, maybe I should also add a DisplacementField image...? But the original example doesn't do that either  🤷